delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/QueryBuilder.scala

```diff
@@ -70,7 +71,9 @@ final case class QueryBuilder private[client] (private val query: JsonObject) {
           params.deprecated.map(term(nxv.deprecated.prefix, _)) ++
           params.rev.map(term(nxv.rev.prefix, _)) ++
           params.createdBy.map(term(nxv.createdBy.prefix, _)) ++
-          params.updatedBy.map(term(nxv.updatedBy.prefix, _)),
+          range(nxv.createdAt.prefix, params.createdAt) ++
```

Comment:
A test for queries with createdAt/updatedAt in QueryBuilderSpec?

Reply:
The time ranges are tested against Elasticsearch, `QueryBuilderSpec` just tests the Json and that does not mean a lot.
It is a bit like testing a generated SQL query and not its result, it does not bring much.